community/tclx/APKBUILD

```diff
@@ -11,11 +11,18 @@ depends="tcl"
 depends_dev="tcl-dev"
 makedepends="tcl-dev"
 subpackages="$pkgname-doc $pkgname-dev"
-source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2"
+source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2
+	rresvport.patch
+	"
 
 _major=${pkgver%.*}
 builddir="$srcdir"/tclx${_major}
 
+prepare() {
+	cd "$builddir"
+	default_prepare
+}
```

Comment:
Why have you added this? `default_prepare` is called by default when custom `prepare` function is not defined.